Duplicate identifier with @types/node 10.0

If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/xxxx` package and had problems.
- [ ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [ ] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @.types/node 10.0

With v9.6.6 everything was fine with my Angular project. Upgraded to 10.0.0 and now I get the following errors running my unit tests:

`
Error: node_modules/@types/node/index.d.ts(2381,15): error TS2300: Duplicate identifier 'URL'.
node_modules/@types/node/index.d.ts(2399,15): error TS2300: Duplicate identifier 'URLSearchParams'.
node_modules/@types/node/index.d.ts(2417,14): error TS2661: Cannot export 'URL'. Only local declarations can be exported from a module.
node_modules/@types/node/index.d.ts(2417,19): error TS2661: Cannot export 'URLSearchParams'. Only local declarations can be exported from a module.
node_modules/typescript/lib/lib.dom.d.ts(12210,11): error TS2300: Duplicate identifier 'URL'.
node_modules/typescript/lib/lib.dom.d.ts(12226,13): error TS2300: Duplicate identifier 'URL'.
node_modules/typescript/lib/lib.dom.d.ts(14282,11): error TS2300: Duplicate identifier 'URLSearchParams'.
node_modules/typescript/lib/lib.dom.d.ts(14309,13): error TS2300: Duplicate identifier 'URLSearchParams'.
`

--- a/types/node/index.d.ts
+++ b/types/node/index.d.ts
@@ -2377,44 +2377,40 @@ declare module "url" {
         unicode?: boolean;
     }
 
-    global {
-        class URL {
-            constructor(input: string, base?: string | URL);
-            hash: string;
-            host: string;
-            hostname: string;
-            href: string;
-            readonly origin: string;
-            password: string;
-            pathname: string;
-            port: string;
-            protocol: string;
-            search: string;
-            readonly searchParams: URLSearchParams;
-            username: string;
-            toString(): string;
-            toJSON(): string;
-        }
-
-        class URLSearchParams implements Iterable<[string, string]> {
-            constructor(init?: URLSearchParams | string | { [key: string]: string | string[] | undefined } | Iterable<[string, string]> | Array<[string, string]>);
-            append(name: string, value: string): void;
-            delete(name: string): void;
-            entries(): IterableIterator<[string, string]>;
-            forEach(callback: (value: string, name: string) => void): void;
-            get(name: string): string | null;
-            getAll(name: string): string[];
-            has(name: string): boolean;
-            keys(): IterableIterator<string>;
-            set(name: string, value: string): void;
-            sort(): void;
-            toString(): string;
-            values(): IterableIterator<string>;
-            [Symbol.iterator](): IterableIterator<[string, string]>;
-        }
+    export class URL {
+        constructor(input: string, base?: string | URL);
+        hash: string;
+        host: string;
+        hostname: string;
+        href: string;
+        readonly origin: string;
+        password: string;
+        pathname: string;
+        port: string;
+        protocol: string;
+        search: string;
+        readonly searchParams: URLSearchParams;
+        username: string;
+        toString(): string;
+        toJSON(): string;
     }
 
-    export { URL, URLSearchParams };
+    export class URLSearchParams implements Iterable<[string, string]> {
+        constructor(init?: URLSearchParams | string | { [key: string]: string | string[] | undefined } | Iterable<[string, string]> | Array<[string, string]>);
+        append(name: string, value: string): void;
+        delete(name: string): void;
+        entries(): IterableIterator<[string, string]>;
+        forEach(callback: (value: string, name: string) => void): void;
+        get(name: string): string | null;
+        getAll(name: string): string[];
+        has(name: string): boolean;
+        keys(): IterableIterator<string>;
+        set(name: string, value: string): void;
+        sort(): void;
+        toString(): string;
+        values(): IterableIterator<string>;
+        [Symbol.iterator](): IterableIterator<[string, string]>;
+    }
 }
 
 declare module "dns" {

